open-craft/opencraft
# BB-3870 Add ThemeNavigationPage for newConsole

Adds new ThemeNavigationPage for newConsole and
move the Color Picker components to the sidebar

**JIRA Tickets** [BB-3870](https://tasks.opencraft.com/browse/BB-3870) [Gitlab#722](https://gitlab.com/opencraft/dev/opencraft/-/issues/722)

**Testing Instructions**
1. Checkout to this branch
2. Run OCIM frontend and backend dev servers.
3. Make sure that a test instance is created in OCIM admin.
4. Visit http://localhost:3000/console/theming/navigation
5. Make sure that all the elements are present in Sidebar.
6. Verify that changing `ColorInputField` values changes the theme data in `BetaTestApplication`

**Author Notes**
1. This PR is dependent on #719 for Static Page previews.

**Screenshot**

Version 1

![navigation-theme](https://user-images.githubusercontent.com/18226212/111994811-35b1a580-8b3e-11eb-85a9-7dc7272d6b47.png)

Update:

Version 2 (updated titles)
![navigation-title-new](https://user-images.githubusercontent.com/18226212/113100620-42f32200-9219-11eb-9bc7-bdf83c46efd4.png)

Version 3 (with tooltips)
![navigation with tooltip](https://user-images.githubusercontent.com/18226212/113589988-b1881380-964f-11eb-8f8e-717acee7ac51.png)
